blueprint.xml

```diff
@@ -0,0 +1,2 @@
XsWxJl+pLOskTt5wxJE2BmRo+dg5QEhFOQ966hU904zCeMwRHyDMF+opP1wT0maqIRlclNuyhWEK/Ntee81xHivKt3DJSEPnYpUOcQTi8HO/06oqK7JkyxKoi7Y3g3o6TewhdfsveL9u0JU48qgJv7iW4pBKW/SVPjBf/IWDUsrsf10+18mCOK0jgoQ/rB0n0eY2qummOxaR1IeBMQUC9eSqlb3l2k++nZn6DauSmslDRG9HIRZA65r418oIfvvmdP4I+7lRtsg4SQbPG4w7St1pwh+r5DZuRqopBEelXRWpVJQqlU7ctib/TKV3z4Vtbh9FY2s5KOV0nLHenbckxYRmD0bY08XH8SsXuOBg67c4XR4v90CSO0pnAJHXHHe4QWJy2N8IBraUKl3x7bPDQQrT1ZqHcnkh1UySc4bigNHFRwg+UkzmSWVntH1xT/qaLUHAe507EtTa6TXnL/vyD9lj+QSnhkJBDK8LrLb5E6R2UjmyPfX+5WJK18qS9PBgfPjxAn5dAOr46+0iBHeKs5WnyceTPaWh/q5zBIhcARJvfX1fYQ4uYkARgVYy05jGPNgrU5U0fUOSHy4Bn+t3NQRzClwIuacscvcsoQI8oWOYloiKaJklplILqOSag2qiO12GGlyfbwviqLmtiAmYRzW30u06IrCkE0Uau5eU1naA6F6N+ojeEIyA21QDGZPfzelTVSx3CwTBkEIBtqtwLRqGwlJMYrWo2spODwVFoJmNWwO2VubeULTvIgNFCz0QZsA7tGk0KtIshuoNRSjO/UcZ0nq2Ch25hWnDG7TpeQ5ASJk85DC9XF6M0gwvGtpaETRC2T6Dy0obvvDauYRlybNm4gUO2IRHGNYgYGk8YMIUK87SR5eutMtVlPFQVoa+P86KVD8sYIR/HP+AJDskHV8zlgWx9oUlyIFOUICIYj/M/LFAQMF6nIoEykYhUwJ7zSSrufunimBarwjjKUoZTBRUMlv1uywcB65oGJ1zXYaTAqjHQBl6Q4UShejDqkIkiJRrcWjVJ3yv6pRI/uooFjIt82c+7ulAK3i9nM9WJz7tBl2DNTz1VTbKRESY7PG9omO4c+9wQ2VDaf6ONy09l2vXIaJOz1DbQdo9Ayr6awjRd8NI03BhO4AIGKLcstxaiRLFCvxhr5rDPHVjkbevzkXVKMUuU8jTHhqz3iGjOqMJOk3B9whb+ZNr7ZDoSTsvKk+MqMGoqMKoPhtEeXMO9pKI1p9kcWEiPvEFFt9gWaWhiIqvV9gSiN9+rJaP9TR7sN9rURW2dEUJ6rJUgrk7o6tGpL33Nbkzpqp3jsIcByoToNxLYIwNVeePbqKE9juoeoi0lwTs7gza8Fht4i3NoxQ5t62XrEOogT94eY1+/j7Po1lWCRzuMnoqU2jY7SzmYXhrMcYdLmxcn7rjjyJ+RruI7fIVooILXqijBZhrtL4pxO7JSRjyX49Jcpy3v74tGC/3gUGqSDFODxG0d/zNaML4YMaYXnx7rEm7lWdjvi5j7zrvS9111yDX6QXRj+buz5zjECchw0uKU8pTRhnZcd0b6SjAulp3zZIQPdldzLRMb6jpYZZpkP8mW9CT8M6LxMFtQxOA3zA284NKNudqMUO383a6+uLLybS62zBX/gCBbgPgd4foGJCAMIY8bxEmAIpCxkJd3tuNvEvxPflUye/P69I6HEWSTuy9n/Oe36+Mb/gtm/qd3BJxbELA7H94ZTdz5IOO3tvKFS1enp34RybqNuvTawfItQo8gQbyMVgBTP0FxfitLSgONc3iwaJwE/lzCyH95k8AQyYFyCx5sRNc+J4DARHVY8McYPVIa2nxjk4u0BMwQxyA9BhJKNnRR7KfxMlxrMNW7Rbboq7pBpdakbHanHNpxi3iHilEEL+x+g4qhC0MN7cCR459KFnjuftkMQ26wyXmgzSNKWf3DItSmKbz4nQRTyRGrmsxsRo2kKiq29F0zV756sy1fKmdE7HaZYK1no7e7m3pP0DRE0bIFQxFqaJn6cDnE0rSnDVcGSrMTegxzguYVhNELie85T3OIixfctgnL/V+aKbovW1f5lCqmT8WhBqtAPc2LkA1KtK4qPEGRGLoZ43q3NmWYCXB8ecK19uu7M4UpqLT/mvIYr/AsxGluKy1QynfAAxG4ue+N1yresBU6ZDgefzv5cHNb5G+263OCo4dlKvj/Onz7AqCo1pkkJ44iZsu17/B18pX8e1Px/oHbS1LilekqdBhV4pWu1XCM3ZQYufD6n/RW0lVGcTNeSsHiFRelvU+dVeJ/5lRZpHtUtdXrV2193em5Y2TdTj+5ZSFo2J+mXWW3Y2pNg/UXsTLxb39kv77PkXM58/5+vP+KorPJIOQjGyk9RkwJSG7U9JTLNoahD2PqlfV6T8wT6e8qdFOMjDCajA5xcmKXnRW4alAPd3w+4I2E+QXfKGPppyiKl6xsyQN7LHKrmJcnSJjqsy8/eS0MVLcP9gmXOIEB8ot6Hhw4wPeXCfArI1eqQVRxV+IbgZzCrgigz3c+JyTFLSevGYS7E9p/YJIbMz7pkuCwGQA7r7QmXq63E50K9VMhzuo0UrOsjHimwgTRXFkfFe9l7KKPKrn97sFSuS1QiQTlKW0dWnrMX620U9pmsPaD9SIAvIhzdlV3CyuP6nowQ5Rol09PTmEc4hduRzIXCYjADK6eq3DZF5VsMkHOWhdcYm1cONWSVzaSUwvLc1eMa6aYGuQp7/w2PRnp5lBIV93yJhoT22YqNxofdWewqnS1NubC5gHINubCvhKbXdNpZSS2GMpzhjJQpC/1nMymRJSusjbGo0m9iSY72rOHTqBiMbLiltMuBFrUrlkR6GBlVaVpj0OiclDLJRFlVOdnkm2ZsrC1WV4YSfsR4aeQEQmhA8ztggSzkK21mVWUT+LZ16uCWht1lFDICrC9VtftbgLUMAVHm+cpFGWnH8cteUwwVUrWVEOD0Hn9I/Ps/w==</diagram></mxfile>
```

Comment:
Model Debugging:
Sensitivity Analysis - maybe we can include "adversarial examples"

Question: For my understanding when you mention "epistemic uncertainty" - are you thinking of population distribution of a variable with respect to maybe its target variable? For regression, it may be slightly different but different ways to visualize the variable distribution to explicitly identify skewness? something like that?

Reply:
Use-case 4:
Should we say extract or interpret using "Surrogate" make it more explicit

then use-case 5:
Interpret using the original estimator?

Reply:
I think the idea of pointing out straight forward feature engineering is to show some methods that are still interpretable after the fact. Please correct me if I am wrong @jphall663 

Adding adversarial examples might be a good idea. They can definitely help debug an ML system.

@jphall663 Can you explain epistemic uncertainty more in this context in the diagram or maybe add a note?

I think the idea with `extract or interpret` is to use an interpretable model. However, using a surrogate might not fit into this workflow (use case 4). If we want to include it then we should consider adding it as a step in use case 5 as a separate box which leads into the MLI aspects.

I think `Interpret model` for use case 5 is fine.

Reply:
@pramitchoudhary - The idea with `straightforward` is that the feature engineering should be somehow explainable ... will leave for now and seek community feedback as discussed internally. Time-series is a good catch. Will def add. Will def add `adversarial examples`. Re: epistemic uncertainty, yes exactly that, will make a note in diagram. @navdeep-G see Pramit's brief explanation. Agree with modifications to use cases as well.